src/frontend/src/optimizer/plan_node/expr_rewritable.rs

```diff
@@ -18,31 +18,40 @@ use paste::paste;
 
 use super::*;
 use crate::expr::ExprRewriter;
-use crate::{for_batch_plan_nodes, for_stream_plan_nodes};
+use crate::for_stream_plan_nodes;
 
 /// Rewrites expressions in a `PlanRef`. Due to `Share` operator,
 /// the `ExprRewriter` needs to be idempotent i.e. applying it more than once
 /// to the same `ExprImpl` will be a noop on subsequent applications.
 pub trait ExprRewritable {
```

Comment:
Add rustdocs to clarify that it only rewrite exprs in a plan, and link to `RewriteExprsRecursive`? 🤔